kit/Kit.cpp

```diff
@@ -157,6 +159,36 @@ static LokHookFunction2* initFunction = nullptr;
 
 namespace
 {
+    // for later consistency checking.
+    static std::string UserDirPath;
+    static std::string InstDirPath;
+
+    std::string pathFromFileURL(const std::string &uri)
+    {
+        std::string decoded;
+        Poco::URI::decode(uri, decoded);
+        if (decoded.rfind("file://", 0) != 0)
```

Comment:
I think this should be compared to `std::string::npos`, otherwise, this will not work for uri that _starts_ with "file://".